fdbserver/BlobMigrator.actor.cpp

```diff
@@ -103,7 +105,7 @@ class BlobMigrator : public NonCopyable, public ReferenceCounted<BlobMigrator> {
 					                         BlobRestorePhase::LOADED_MANIFEST));
 					return Void();
 				}
-			} else if (phase >= BlobRestorePhase::COPYING_DATA) {
+			} else if (phase >= BlobRestorePhase::COPYING_DATA && phase < BlobRestorePhase::DONE) {
```

Comment:
if restore is done and blob migrator is restarted, it's not an error case.. nothing need to be done